blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -215,4 +217,9 @@ const selectors = createSelector(
     [runSelector, isMultiBranchSelector, previousSelector],
     (run, isMultiBranch, previous) => ({ run, isMultiBranch, previous }));
 
-export default translate(['jenkins.plugins.blueocean.dashboard.Messages'], { wait: true })(connect(selectors, actions)(RunDetails));
+const composed  = compose(
```

Comment:
even less clear what this does now cc @tfennelly> Went from boiler plate to magical more verbose boilerplate. 👎🏻